Shoko.Server/API/v3/Controllers/SeriesController.cs

```diff
@@ -2445,6 +2445,8 @@ public async Task<ActionResult> PostSeriesUserVote([FromRoute, Range(1, int.MaxV
             _ => series.AniDB_Anime?.GetFinishedAiring() ?? false ? AniDBVoteType.Anime : AniDBVoteType.AnimeTemp,
         };
         await _seriesService.AddSeriesVote(series, voteType, vote.GetRating());
+        var pluginVoteType = voteType == AniDBVoteType.Anime ? VoteType.Permanent : VoteType.Temporary;

```

Comment:
This should either be moved into the series service, or the method on the series service should be moved to the user data service and  exposed so other plugins can use it (in addition to the core) to vote on series.